packages/sanity/src/structure/panes/documentList/__tests__/useDocumentSheetColumns.test.ts

```diff
@@ -20,7 +21,7 @@ describe('useDocumentSheetColumns', () => {
         {name: 'email', type: {name: 'string'}},
         {name: 'age', type: {name: 'number'}},
         {
-          name: 'address',
+          name: 'address.place',
```

Comment:
This type of schema name won't be allowed in the studio, it doesn't support dot notation in the naems
<img width="518" alt="Screenshot 2024-05-21 at 08 11 35" src="https://github.com/sanity-io/sanity/assets/46196328/ff791ec0-c0a6-4ba0-a7a4-750de50b49a7">

Reply:
- [ ] tests don't need to test with dot notation